mobile/screens/onboarding/RequestItems.tsx

```diff
@@ -233,7 +218,18 @@ export default function RequestedItems({
               <TouchableOpacity
                 style={[styles.button, { width: 350 }]}
                 onPress={() => {
-                  setModalVisible(!modalVisible);
+                  if (
+                    itemsCount[
+                      settings.items?.findIndex(
+                        (item) => item.itemName === "carSeat"
+                      ) as number
```

Comment:
idt you need `as number` — findIndex should return a number, right?

I guess it could be `null`, but you might want to check for that separately. NBD tho.